apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts

```diff
@@ -0,0 +1,112 @@
+import { CreateBookingInput } from "@/ee/bookings/inputs/create-booking.input";
+import { CreateReccuringBookingInput } from "@/ee/bookings/inputs/create-reccuring-booking.input";
+import { GetUser } from "@/modules/auth/decorators/get-user/get-user.decorator";
+import { AccessTokenGuard } from "@/modules/auth/guards/access-token/access-token.guard";
+import {
+  Controller,
+  Post,
+  Logger,
+  UseGuards,
+  Req,
+  InternalServerErrorException,
+  Body,
+  HttpException,
+} from "@nestjs/common";
+import { User } from "@prisma/client";
+import { Request } from "express";
+import { NextApiRequest } from "next/types";
+
+import { SUCCESS_STATUS } from "@calcom/platform-constants";
+import {
+  handleNewBooking,
+  BookingResponse,
+  HttpError,
+  handleNewRecurringBooking,
+  handleInstantMeeting,
+} from "@calcom/platform-libraries";
+import { ApiResponse } from "@calcom/platform-types";
+
+const handleBookingErrors = (err: Error | HttpError | unknown, type?: "recurring" | `instant`): void => {
+  const errMsg = `Error while creating ${type ? type + " " : ""}booking.`;
+  if (err instanceof HttpError) {
+    const httpError = err as HttpError;
+    throw new HttpException(httpError?.message ?? errMsg, httpError?.statusCode ?? 500);
+  }
+
+  if (err instanceof Error) {
+    const error = err as Error;
+    throw new InternalServerErrorException(error?.message ?? errMsg);
+  }
+
+  throw new InternalServerErrorException(errMsg);
+};
+
+@Controller({
+  path: "ee/bookings",
+  version: "2",
+})
+@UseGuards(AccessTokenGuard)
+export class BookingsController {
+  private readonly logger = new Logger("ee bookings controller");
+
+  @Post("/create")
+  async createBooking(
+    @Req() req: Request & { userId?: number },
+    @Body() _: CreateBookingInput,
```

Comment:
`@Body() _: CreateBookingInput` - if it is not used, shouldn't we remove all occurrences of `@Body() _: CreateBookingInput` in the controllers?

Reply:
how do we specify that we want the body to be validated  if we dont put this ?